src/main/java/core/basesyntax/dao/StorageDaoImpl.java

```diff
@@ -22,4 +22,13 @@ public User get(String login) {
         }
         return null;
     }
+
+    @Override
+    public void deleteByLogin(String login) {
+        User user = get(login);
+        if (user != null) {
+            Storage.people.remove(user);
+        }
+    }
```

Comment:
The `deleteByLogin` method correctly removes a user by login. However, ensure that the `get` method handles cases where the user might not be found to avoid potential `NullPointerException`. Consider using `Optional<User>` for safer handling of absent users.